src/blot/scroll.ts

```diff
@@ -23,13 +22,10 @@ class ScrollBlot extends ParentBlot implements Root {
   public static scope = Scope.BLOCK_BLOT;
   public static tagName = 'DIV';
 
-  public registry: Registry;
   public observer: MutationObserver;
 
-  constructor(registry: Registry, node: HTMLDivElement) {
-    // @ts-expect-error
-    super(null, node);
-    this.registry = registry;
+  constructor(public registry: Registry, node: HTMLDivElement) {
+    super(null as any /* scroll is the root with no parent */, node);
```

Comment:
I'd do `// @ts-expect-error scroll is the root with no parent` and avoid `as` so we can be notified if `super` accepts `null`.